tuplex/core/include/logical/SortOperator.h

```diff
@@ -0,0 +1,45 @@
+//
+// Created by Colby Anderson on 5/27/21.
+//
+
+#ifndef TUPLEX_SORTOPERATOR_H
+#define TUPLEX_SORTOPERATOR_H
+
+#include "LogicalOperator.h"
+
+namespace tuplex {
+    class SortOperator : public LogicalOperator {
+    private:
+        std::vector<size_t> _order;
+        std::vector<size_t> _orderEnum;
+        std::vector<std::string> _columns;
+    public:
+        SortOperator(LogicalOperator *parent, std::vector<size_t> order, std::vector<size_t> orderEnum);
+
+        std::vector<size_t> orderEnum() { return _orderEnum; }
+
+        std::vector<size_t> order() { return _order; }
+
+        virtual std::string name() override { return "sort"; }
```

Comment:
should be const too? if not, probably should change across all operators (if so, please do in separate commit so it's clear this change has been done when squashing the PR).